OctoAwesome/OctoAwesome/Simulation.cs

```diff
@@ -155,26 +160,26 @@ public void ExitGame()
         /// <param name="entity">Neue Entity</param>
         public void AddEntity(Entity entity)
         {
-            if (entity == null)
-                throw new ArgumentNullException(nameof(entity));
+            if (entity == null) throw new ArgumentNullException(nameof(entity));
```

Comment:
CodeStyle!

Reply:
Der Coding Style tut mir in den Augen weh. 
Aber bitte. Ich versuch mich dran zu halten.

Reply:
Wir haben ihn in der Gemeinschaft mit Abstimmung entschieden. Auch ich, auch lassi, Jvbsl und wer nicht alles müssen sich daran halten selbst wenn jeder von uns eigene coding style vorlieben hatt